posthog/batch_exports/service.py

```diff
@@ -372,3 +409,39 @@ async def updater(_: ScheduleUpdateInput) -> ScheduleUpdate:
     return await handle.update(
         updater=updater,
     )
+
+
+def create_batch_export_backfill(
+    batch_export_id: UUID,
+    team_id: int,
+    start_at: str,
+    end_at: str,
+    status: str = BatchExportRun.Status.RUNNING,
+):
+    """Create a BatchExportBackfill.
+
+    Args:
+        data_interval_start:
+        data_interval_end:
```

Comment:
nit: It looks like these were renamed, and there's more args (although they all seem pretty self explanatory).

Reply:
Yeah, I'll adjust this docstring to match the current function signature.

Reply:
Also updated a couple others. Thank you!